packages/botonic-plugin-contentful/tests/tools/l10n/csv-export.test.ts

```diff
@@ -34,6 +35,21 @@ test('TEST: ContentToCsvLines.getCsvLines Text', () => {
   ])
 })
 
+test('TEST: ContentToCsvLines.getCsvLines URL', () => {
```

Comment:
It would be nice to add a test for URLS with a /es/ in its URL and confirm that PostProcessor is working corrrectly

Reply:
yes, but run out of time :-(